plotly/colors.py

```diff
@@ -174,17 +177,18 @@ def color_parser(colors, function):
             return new_color_list
 
 
-def validate_colors(colors):
+def validate_colors(colors, colors_list=None):
```

Comment:
It's to initialize colors_list. I ran into difficulties with _not_ doing that in convert_colors_to_same_type so I added `colors_list=None` to that function as well as the `validate_colors` one. Should it not be there?

Or wait, it's 🎎  so do you just want documentation on it?